### PR TITLE
Remove durable flows

### PR DESCRIPTION
**Pull request by pavelgj**

### CODE REVIEW

**Comment by debkanchan** [comment]
why tho?

**Comment by pavelgj** [comment]
> why tho?

We want to simplify the core framework, durable flows add a lot of complexity. We are looking to support stateful agentic use cases via a more focused framework rather than via general purpose workflow engine.

File: js/flow/src/flow.ts [hunk 1]
```diff
@@ -790,43 +397,38 @@ export class Flow<
...
-        const state = await this.runDirectly(input, {
+        const result = await this.invoke(input, {
           streamingCallback: ((chunk: z.infer<S>) => {
             res.write(JSON.stringify(chunk) + streamDelimiter);
           }) as S extends z.ZodVoid ? undefined : StreamingCallback<z.infer<S>>,
           auth,
         });
-        res.write(JSON.stringify(state.operation));
+        res.write({
+          result: result.result, // Need more results!!!!
```
Comment (apascal07) [inline js/flow/src/flow.ts:407]: ?

File: js/flow/src/index.ts [hunk 2]
```diff
@@ -14,26 +14,18 @@
...
-export { FirestoreStateStore } from './firestoreStateStore.js';
 export {
   Flow,
   defineFlow,
   defineStreamingFlow,
   startFlowsServer,
   type CallableFlow,
   type FlowAuthPolicy,
-  type StepsFunction,
+  type FlowFn as StepsFunction,
```
Comment (apascal07) [inline js/flow/src/index.ts:24]: `type FlowFn,`

**Review by apascal07 — APPROVED** [review]
End of an era. 🥲

File: js/flow/src/flow.ts (hunk 1 above)
Comment (pavelgj) [reply js/flow/src/flow.ts:407]: `result: result.result`... joke?! I try...

Comment (apascal07) [reply js/flow/src/flow.ts:407]: Oh I saw that and was like jeeze how many more are you expecting? Didn't get it was a meta-joke. :-)

**Pull request merged**